cli/atlas/formatting.go

```diff
@@ -40,17 +41,23 @@ func ServerURL(str string) (string, error) {
 }
 
 // ProjectRoot determines the root directory of an application. The project
-// root is considered to be anything after go/src/...
-func ProjectRoot(dirString string) (string, error) {
-	dirs := strings.Split(dirString, "/")
-	for i, dir := range dirs {
-		if strings.ToLower(dir) == "go" && i+1 < len(dirs) {
-			if i+2 < len(dirs) && strings.ToLower(dirs[i+1]) == "src" {
-				return strings.Join(dirs[i+2:], "/"), nil
-			}
-		}
+// root is considered to be anything in $GOPATH/src or below
+func ProjectRoot(gopath, workdir string) (string, error) {
+	if gopath == "" {
```

Comment:
technically, if no gopath is provided you could get it yourself: https://stackoverflow.com/a/32650077/437456